cli/lib/kontena/cli/stacks/stack_name.rb

```diff
@@ -4,7 +4,7 @@ module Kontena::Cli::Stacks
   class StackName
     # A class for parsing stack name strings, such as kontena/foo:1.0.0
 
-    attr_reader :user, :stack, :version
+    attr_accessor :user, :stack, :version
```

Comment:
Why?

Reply:
So that it's possible to `x = StackName.new('user/stack'); x.version = '1.2.3'`.

Reply:
I don't see that setter is needed because version can be passed via `.new`?